net/happygears/proto/NSGPythonSDK.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package net.happygears.proto.nsg_python_sdk;
+import "net/happygears/proto/NSG.proto";
+
+message ServiceInfo {
+  string script = 1;
+  string port = 2;
+}
+
+message StatusResp {
+  string status = 1;
+}
+
+message TagsAppRequest {
+  nsg.Device device = 1;
+}
+
+message KeyedTagList {
+  repeated string tags = 1;
+  bool is_device = 2;
+  bool is_interface = 3;
+  nsg.DataSource ds = 4;
+  int64 index = 5;
+  string monitoring_variable_hint = 6;
+}
+
+message TagUpdateJournal {
+  repeated KeyedTagList added = 1;
+  repeated KeyedTagList removed = 2;
+}
+
+message VariablesAppRequest {
+  nsg.Device device = 1;
+}
+
+message PollingVariable {
```

Comment:
this appears to be a copy of the same protobuf message from NSG. Lets try to avoid copying

Reply:
Yes, we can use PollingVariable message from NSG. 

I see couple of fields are not present in NSG PollingVariable message. 

- nsg.DataSource dsc
- string metric_name

Can we add them to the NSG (proto-oss) to avoid copying message?

Reply:
yes go ahead and add those fields. Looks like NSG.PollingVariable is used only with the following protobuffers in NSG right now:

`message PollingVariables {
    repeated PollingVariable variables = 1;
}

message PollingVariableMap {
    option deprecated = true;

    map <string, PollingVariables> polling_variables = 1;

    repeated LogRecord log_records = 99;
}
`

but these protobuffers are not used anywhere. Please mark them as deprecated when you edit NSG.proto, we'll remove them later

Reply:
NSGPython proto uses PollingVariables message as part of `VariablesAppResponse `. Can we mark both `NSGPython.PollingVariable` and `NSGPython.PollingVariables` as deprecated and use NSG.PollingVariable and NSG.PollingVariables. This will avoid copying `NSGPython.PollingVariables` to `NSGPythonSDK` ?

Reply:
yes, makes sense. Lets move both to NSG and mark both of them as deprecated in NSGPython